test/spec/util/tile-error-collection.spec.js

```diff
@@ -0,0 +1,196 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var _ = require('underscore');
+var TileErrorCollection = require('../../../src/util/tile-error-collection');
+
+describe('util/tile-error-collection', function () {
+  var interceptAjaxCall;
+  beforeEach(function () {
+    interceptAjaxCall = null;
+
+    this.error = { type: 'limit', message: 'Some message' };
+    this.tiles = [
+      new Backbone.Model({ url: 'some_url/0/0/0/1.png', tileDomNode: {} }),
+      new Backbone.Model({ url: 'some_url/0/0/0/2.png', tileDomNode: {} }),
+      new Backbone.Model({ url: 'some_url/0/0/0/3.png', tileDomNode: {} })
+    ];
+    this.collection = new TileErrorCollection(this.tiles);
+
+    this.originalAjax = $.ajax;
+    $.ajax = function (params) {
+      interceptAjaxCall && interceptAjaxCall(params);
+      return {
+        complete: function (cb) {
+          cb();
+          return this;
+        },
+        success: function (cb) {
+          cb();
+          return this;
+        },
+        error: function (cb) {
+          cb();
+          return this;
+        }
+      };
+    };
+  });
+
+  describe('._onAdd', function () {
+    var tile = {
+      url: 'some_url/0/0/0/4.png',
+      tileDomNode: {}
+    };
+
+    it('should set the error tile overlay when a new model is added', function () {
+      spyOn(document.body, 'contains').and.returnValue(true);
+      this.collection.add(tile);
+      var model = _.last(this.collection.models);
+      expect(model.get('tileDomNode').src.indexOf('data:image/svg+xml;base64,')).not.toBe(-1);
+    });
+
+    it('should add the model to the qeue if is not there', function () {
```

Comment:
Perhaps worth adding a `describe('when running is true'...` to keep the test more consistent?